### PR TITLE
Updated README to clairfy use kong 1.5.1 for OCP4

### PR DESCRIPTION
**Pull request by gyliu513**
Fixed https://github.com/Kong/kong-operator/issues/1

Please refer to https://github.com/Kong/kong/issues/5708 for detail.

FYI @hutchic

### CODE REVIEW

**Comment by hbagdi** [comment]
This should be configurable using custom values already.

We shouldn't merge this in:
1. This operator is significantly out of date at this point and doing these changes are hard to verify and ship to Redhat's registry.
2. Kong's Helm Chart has seen significant changes, which need to be included here at this point.

**Comment by gyliu513** [comment]
Thanks @hbagdi and @hutchic for the discussion here.

Question to @hbagdi , does Kong has some plan use Go operator or you will continue to use this helm operator?

For this PR, I think we may need to update the README of this operator to mention that we should use 1.5.1 kong for OCP, comments?

**Comment by hbagdi** [comment]
> does Kong has some plan use Go operator or you will continue to use this helm operator?

Currently we will use helm operator but we do have plans to move over to Go Operator in the long-term.

> For this PR, I think we may need to update the README of this operator to mention that we should use 1.5.1 kong for OCP, comments?

Sure. We want to hold off releases for now but doc improvements are okay.

**Comment by gyliu513** [comment]
@hbagdi done.

File: README.md [hunk 1]
```diff
@@ -2,3 +2,8 @@
 Kong operator follows the Kubernetes Operator pattern to provide a way
 to deploy and manage Kong clusters inside a Kubernetes cluster.
 It can be used on both vanilla Kubernetes and OpenShift.
+
+# NOTE
+If you are using OCP4, please update the kong image to `kong:1.5.1`.
+For more detail, please refer to issue [Kong failed to start on OCP4.3](
```
Comment (hbagdi) [inline README.md:8]: Let's change the link test to [issue 5708].

Comment (gyliu513) [reply README.md:8]: Done

**Pull request merged**